tests/unit/models/360-core.js

```diff
@@ -127,9 +164,11 @@ describe("SerSol 360 Core Model >", function() {
     });
 
     it("should not enhance a journal search result without an issn or eissn", function() {
-      expect(titles.length).toEqual(2);
+      expect(titles.length).toEqual(4);
       expect(titles[0].title).toEqual("The New England journal of medicine");
       expect(titles[1].title).toEqual("The New England journal of medicine and surgery");
+      expect(titles[2].title).toEqual("Cell");
+      expect(titles[3].title).toEqual("Art");
```

Comment:
How do we confirm a journal has _not_ been enhanced here? Should we be `expect`ing a property to not exist?

Reply:
Un-enhanced Journals won't have the `shouldEnhance` property and won't be added to the list of Journals to enhance. Added a test for the same.